packages/enzyme/package.json

```diff
@@ -33,6 +33,9 @@
   ],
   "author": "Leland Richardson <leland.richardson@airbnb.com>",
   "license": "MIT",
+  "engines": {
+    "node": ">=5"
```

Comment:
I don't think we should specify this (and yarn shouldn't be enforcing it anyway - npm does not).

I think @aweary should remove or update "engines" on rst-selector-parser instead.